CONTRIBUTING.md

```diff
@@ -25,22 +25,22 @@ See the [SMQTK README] file for additional information.
 
         $ git push origin HEAD -u
 
-5.  Visit the Kitware SMQTK page [on GitHub], browse to the "Pull requests" tab
+5.  Visit the Kitware SMQTK-Core page [on GitHub], browse to the "Pull requests" tab
     and click on the "New pull request" button in the upper-right.
     Click on the "compare across forks" link, browse to your fork and browse to
     the topic branch to submit for the pull request.
     Finally, click the "Create pull request" button to create the request.
 
 
-SMQTK uses GitHub for code review and [Travis-CI] for continuous testing as new
+SMQTK-Core uses GitHub for code review and [Travis-CI] for continuous testing as new
 pull requests are made.
 All checks/tests must pass before a PR can be merged.
 
 Sphinx is used for manual and automatic API [documentation].
 
 
 [SMQTK README]: README.md
-[on GitHub]: https://github.com/Kitware/SMQTK
-[build instructions]: http://smqtk.readthedocs.org/en/latest/building.html
+[on GitHub]: https://github.com/Kitware/SMQTK-Core
+[build instructions]: https://smqtk-core.readthedocs.io/en/stable/installation.html
 [Travis-CI]: https://travis-ci.org/Kitware/SMQTK/
-[documentation]: docs/
+[documentation]: https://smqtk-core.readthedocs.io/en/stable/
```

Comment:
I'm both OK with this as well as have a desire to point to local documentation since RTD will obviously not reflect a contributor's work. Maybe update/add to the above sphinx docs line to refer to the README.md file for locally building documentation? What are your opinions on this?

Reply:
I think what I'm going to do is revert this to the local docs and then just add a RTD link to the README.md if thats ok

Reply:
I think that's probably fine.